am-lib/src/main/java/uk/gov/hmcts/reform/amlib/AccessManagementService.java

```diff
@@ -45,21 +50,36 @@ public void createResourceAccess(ExplicitAccessRecord explicitAccessRecord) {
     }
 
     /**
-     * Returns `resourceJson` when record with userId and resourceId exist and has READ permissions, otherwise null.
+     * Returns FilterResourceResponse when record with userId and resourceId exist and has READ permissions,
```

Comment:
Link class in Javadoc if you mention class name.

See: https://docs.oracle.com/javase/7/docs/technotes/tools/windows/javadoc.html#link

Reply:
👍